core/lib/src/main/java/org/opennms/core/utils/SystemInfoUtils.java

```diff
@@ -40,12 +40,18 @@
 public class SystemInfoUtils {
     private static final Logger LOG = LoggerFactory.getLogger(SystemInfoUtils.class);
 
+    public static final String OPENNMS_INSTANCE_ID_SYS_PROP = "org.opennms.instance.id";
+    public static final String DEFAULT_INSTANCE_ID = "OpenNMS";
```

Comment:
Long-term, this part of the model should be merged into the OnmsMonitoringSystem that represents the OpenNMS system.